main.c

```diff
@@ -392,12 +392,18 @@ void update_info(void)
 	}
 }
 
-int ptr_third_x(void)
+int nav_button(void)
 {
-	int x, y;
+	int x, y, nav_width;
```

Comment:
Aren't we shadowing the config var `NAV_WIDTH` here? I know C is case sensitive but it doesn't feel right to have two vars with same name. I would suggest using something like `nw`

Reply:
Refer to https://github.com/nsxiv/nsxiv/pull/155#discussion_r741560941

I agree with your concern, I'd be glad to switch back, or change to another name, since @TAAPArthur thinks `nw` is not good in readability.

Reply:
I think short names are okay for local vars. But if @TAAPArthur has a problem with `nw` then maybe we can use something else. Maybe `nav_w` , seems clear enough. My main concern is to not have the same name as `NAV_WIDTH`.

Reply:
I changed back to `nw`, it's pretty easy to know it's meaning from the context.

Reply:
Feel like we should've waited for @TAAPArthur 's response. But if he still has a problem then we can just use `nav_w`. Should take care of both his and my concern.

Reply:
I just like being more explicit with variable names. I think width would be better than nav_w which I think is better than nw.

Not gonna hold the PR on it. Once you know what it means it becomes obvious, but I first looked at this and went "what does nw mean" for like 2s.

Reply:
`width` can be confused with the window width. I think `nav_w` is more disambiguous. But in this case don't think it should take long to figure out what `nw` means given how short the function is.